src/common/workspace-store.ts

```diff
@@ -7,10 +7,12 @@ import { broadcastMessage } from "../common/ipc";
 import logger from "../main/logger";
 
 export type WorkspaceId = string;
+export type ClusterId = string;
```

Comment:
This should be imported from some place else

Reply:
Fixed

Reply:
It's already defined in `cluster-store.ts` as:
```
export type ClusterId = string;
```
So just `import type {ClusterId} from "../cluster-store.ts"`